app/models/user.rb

```diff
@@ -365,7 +365,7 @@ def military_information
 
   def veteran_status
     @veteran_status ||= if Flipper.enabled?(:veteran_status_updated)
-                          VAProfile::VeteranStatus::Service.new(self)
```

Comment:
Will we need to delete the service defined here? It doesn't look like it's referenced elsewhere.

Reply:
The VAProfileRedis::VeteranStatus uses that service class under the hood. 

We switch to using the VAProfileRedis::VeteranStatus because we've moved to a caching solution. If the cache data isn't found VAProfileRedis::VeteranStatus then calls VAProfile::VeteranStatus::Service.new under the hood